README.md

```diff
@@ -67,6 +67,21 @@ app's Gemfile.
 
 ## Usage
 
+### Configuration
+
+Journaling provides a number of different configuation options that can be set in Ruby using an initializer. Those values are:
+
+#### `Journaled.default_app_name`
+
+  This is described in the proceeding paragraph and is used to specify which app name to use, which corresponds to which Journaled Stream to send events too.
```

Comment:
If we're going to document this we should say that it will be overrridden by the model, however that works, I forget.